tests/e2e/README.md

```diff
@@ -88,6 +85,8 @@ Setup Wizard e2e test (located in `activate-and-setup` directory) will run befor
 
 - `git checkout master` or checkout the branch where you need to run tests 
 
+- Run `nvm use`
+
 - Run `npm install`
 
 - Run `npm install jest --global`
```

Comment:
Thanks for pointing it out @ivatautomattic! I created an issue so that it doesn't get lost since this PR is already approved: https://github.com/woocommerce/woocommerce/issues/27764.